script/drupal-aws-cache.js

```diff
@@ -121,5 +121,8 @@ async function createCacheFile() {
 if (options.fetch) {
   fetchCache();
 } else {
-  createCacheFile();
+  createCacheFile().catch(error => {
+    console.error(`Error in createCacheFile: ${error}`);
+    process.exit(1);
+  });
```

Comment:
`createCacheFile` is an async function. With the current version of node, failures in this would be returned as an unhandled promise and exit with the process with zero (success). 

This caused the `Cache Drupal Content` job to succeed on [this](http://jenkins.vfs.va.gov/job/testing/job/vets-website/job/master/8809/console) build which caused a difficult to trace bug further downstream in the CI pipeline. 

This catches the error and exits with a non-zero (failure) value.